### DISCUSSION TITLE
닉네임 모양 이대로 괜찮은가요 ? 'ㅇ'

### DISCUSSION THREAD

**Discussion by ghost** [tastekim/WeAllLie-BE#38] [Show and tell]
<img width="530" alt="스크린샷 2022-11-13 오후 7 05 08" src="https://user-images.githubusercontent.com/112525911/201516411-b40f6250-7127-4cbf-88b5-a2b24da5a454.png">

닉네임 이렇게 만들었는데 제대로 된 모양인지 모르겠어요
혹시 _ 없이가 낫겠다든지, 다 대문자가 낫겠다든지 하는거 있으면 말씀해주세요 ;ㅁ ;

**Comment by tastekim** [comment]
전 괜찮은거 같아요 !

**Reply by ghost** [reply]
넵!!
